tests/Agent/IntegrationTests/ContainerApplications/AwsSdkTestApp/AwsSdkExercisers/AwsSdkDynamoDBExerciser.cs

```diff
@@ -0,0 +1,250 @@
+        private readonly AmazonDynamoDBClient _amazonDynamoDBClient;
+
+        public AwsSdkDynamoDBExerciser()
+        {
+            _amazonDynamoDBClient = GetDynamoDBClient();
+        }
+        
+
+        private AmazonDynamoDBClient GetDynamoDBClient()
+        {
+
+            AmazonDynamoDBConfig clientConfig = new AmazonDynamoDBConfig();
+            // Set the endpoint URL
+            clientConfig.ServiceURL = "http://dynamodb:8123"; // port must match what is set in docker compose
+            clientConfig.AuthenticationRegion = "us-west-2";
+            var creds = new BasicAWSCredentials("xxx", "xxx");
+            AmazonDynamoDBClient client = new AmazonDynamoDBClient(creds, clientConfig);
+
+            return client;
+        }
+
+        #region Table Operations
+        [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
+        public async Task<bool> CreateTableAsync(string name)
+        {
+            var response = await _amazonDynamoDBClient.CreateTableAsync(new CreateTableRequest
+            {
+                TableName = name,
+                AttributeDefinitions = new List<AttributeDefinition>()
+                {
+                    new AttributeDefinition
+                    {
+                        AttributeName = "title",
+                        AttributeType = ScalarAttributeType.S,
+                    },
+                    new AttributeDefinition
+                    {
+                        AttributeName = "year",
+                        AttributeType = ScalarAttributeType.N,
+                    },
+                },
+                KeySchema = new List<KeySchemaElement>()
+                {
+                    new KeySchemaElement
+                    {
+                        AttributeName = "year",
+                        KeyType = KeyType.HASH,
+                    },
+                    new KeySchemaElement
+                    {
+                        AttributeName = "title",
+                        KeyType = KeyType.RANGE,
+                    },
+                },
+                ProvisionedThroughput = new ProvisionedThroughput
+                {
+                    ReadCapacityUnits = 5,
+                    WriteCapacityUnits = 5,
+                },
+            });
+            if (response.HttpStatusCode != System.Net.HttpStatusCode.OK)
+            {
+                Console.WriteLine($"Got bad http status code: {response.HttpStatusCode}");
+            }
+
+            // Wait until the table is ACTIVE and then report success.
+            Console.Write("Waiting for table to become active...");
+
+            var request = new DescribeTableRequest
+            {
+                TableName = response.TableDescription.TableName,
+            };
+
+            TableStatus status;
+
+            int sleepDuration = 2000;
+
+            var startTime = DateTime.Now;
+            do
+            {
+                Thread.Sleep(sleepDuration);
```

Comment:
Since you're already `async`,
```suggestion
                await Task.Delay(sleepDuration);
```